src/course-updates/course-update/CourseUpdate.jsx

```diff
@@ -18,6 +20,12 @@ const CourseUpdate = ({
     <div className="course-update" data-testid="course-update">
       <div className="course-update-header">
         <span className="course-update-header__date small font-weight-bold">{dateForUpdate}</span>
+        {!isDateForUpdateValid(dateForUpdate) && (
+          <div className="course-update-header__error">
```

Comment:
I don't know what this error looks like, but does it make sense to use the Alert component?

Reply:
no